packages/grafana-ui/src/components/ConfirmButton/DeleteButton.tsx

```diff
@@ -22,7 +26,25 @@ export const DeleteButton: FC<Props> = ({ size, disabled, onConfirm, 'aria-label
       disabled={disabled}
       onConfirm={onConfirm}
     >
-      <Button aria-label={ariaLabel} variant="destructive" icon="times" size={size || 'sm'} />
+      <Button
+        type={'button'}
+        aria-label={ariaLabel}
+        variant="secondary"
+        fill={'text'}
+        icon={'trash-alt'}
+        size={size || 'sm'}
+        className={styles.button}
+      />
     </ConfirmButton>
   );
 };
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    button: css`
+      &:hover {
+        color: ${theme.colors.error.main};
```

Comment:
Rather than adding custom styles could we set the Button prop `variant="destructive"`?

Reply:
I thought we had some prettier or lint rule that enforced those? Maybe that have changed 🤔